README.md

```diff
@@ -1 +1,285 @@
-# clinic_db
\ No newline at end of file
+# curriculum-databases-projects-template
+
+> This template should be used for database related projects at Microverse.
+> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+
+
```

Comment:
- Optional. I would recommend removing this part form your README file to keep it clean.